orttraining/orttraining/python/training/ortmodule/torch_cpp_extensions/cuda/torch_gpu_allocator/setup.py

```diff
@@ -25,7 +25,9 @@
 
 extra_compile_args = {"cxx": ["-O3"]}
 if not use_rocm:
-    extra_compile_args.update({"nvcc": os.environ["ONNXRUNTIME_CUDA_NVCC_EXTRA_ARGS"].split(",")})
```

Comment:
curious is anything wrong writing this way?

Reply:
when `ONNXRUNTIME_FORCE_CUDA=1`, there is no `ONNXRUNTIME_CUDA_NVCC_EXTRA_ARGS` in the env, because there is no cuda version set on the ort metadata file

Reply:
I see. thanks.